web/src/user_groups.ts

```diff
@@ -240,6 +240,10 @@ export function get_realm_user_groups_for_dropdown_list_widget(
         return [];
     }
 
+    if (setting_name === "resolve_topic_group") {
+        group_setting_config = {...group_setting_config, allow_nobody_group: false};
+    }
+
```

Comment:
This is done to remove the "role:nobody" from dropdown options in case of `resolve_topic_group` since it is represented by checkbox.